### PR TITLE
Add latest template files

### PR DESCRIPTION
**Pull request by mariux**
- Update CODEOWNERS file to latest template
- Update CONTRIBUTING.md to latest template
- Add test/README.md from template
- Import adjusted README.md template
- Add CHANGELOG.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,217 @@
...
+[homepage]: https://mineiros.io/?ref=terraform-aws-vpc
+[hello@mineiros.io]: mailto:hello@mineiros.io
+
+[badge-build]: https://mineiros.semaphoreci.com/badges/terraform-aws-vpc/branches/master.svg?style=shields&key=df11a416-f581-4d35-917a-fa3c2de2048e
+[badge-semver]: https://img.shields.io/github/v/tag/mineiros-io/terraform-aws-vpc.svg?label=latest&sort=semver
+[badge-license]: https://img.shields.io/badge/license-Apache%202.0-brightgreen.svg
+[badge-terraform]: https://img.shields.io/badge/terraform-0.13%20and%200.12.20+-623CE4.svg?logo=terraform
+[badge-slack]: https://img.shields.io/badge/slack-@mineiros--community-f32752.svg?logo=slack
+
+<!-- markdown-link-check-disable -->
```
Comment (soerenmartius) [inline README.md:197]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,72 +1,217 @@
...
+[build-status]: https://mineiros.semaphoreci.com/projects/terraform-aws-vpc
+[releases-github]: https://github.com/mineiros-io/terraform-aws-vpc/releases
+<!-- markdown-link-check-enable -->
```
Comment (soerenmartius) [inline README.md:200]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,72 +1,217 @@
...
+
+[releases-terraform]: https://github.com/hashicorp/terraform/releases
+[apache20]: https://opensource.org/licenses/Apache-2.0
+[slack]: https://join.slack.com/t/mineiros-community/shared_invite/zt-ehidestg-aLGoIENLVs6tvwJ11w9WGg
+[Terraform]: https://www.terraform.io
+[AWS]: https://aws.amazon.com/
+[Semantic Versioning (SemVer)]: https://semver.org/
+
+<!-- markdown-link-check-disable -->
```
Comment (soerenmartius) [inline README.md:209]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,72 +1,217 @@
...
+[variables.tf]: https://github.com/mineiros-io/terraform-aws-vpc/blob/master/variables.tf
+[examples/]: https://github.com/mineiros-io/terraform-aws-vpc/blob/master/examples
+[Issues]: https://github.com/mineiros-io/terraform-aws-vpc/issues
+[LICENSE]: https://github.com/mineiros-io/terraform-aws-vpc/blob/master/LICENSE
+[Makefile]: https://github.com/mineiros-io/terraform-aws-vpc/blob/master/Makefile
+[Pull Requests]: https://github.com/mineiros-io/terraform-aws-vpc/pulls
+[Contribution Guidelines]: https://github.com/mineiros-io/terraform-aws-vpc/blob/master/CONTRIBUTING.md
+<!-- markdown-link-check-enable -->
```
Comment (soerenmartius) [inline README.md:217]:
```suggestion
```

File: test/README.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+2. Install [Terraform].
+3. Set your AWS credentials as environment variables: `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`
+4. Install go dependencies: `go mod download`
+5. Run all tests: `go test -v -count 1 -timeout 45m -parallel 128 ./test/...`
+   or use the convenient `make test/unit-tests` Makefile target.
+6. Run a specific test: `go test -count 1 -v -timeout 45m -parallel 128 test/example_test.go`
+
+<!-- References -->
+
+<!-- markdown-link-check-disable -->
```
Comment (soerenmartius) [inline test/README.md:66]:
```suggestion
```

File: test/README.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+[Makefile]: https://github.com/mineiros-io/terraform-aws-vpc/blob/master/Makefile
+[Testdirectory]: https://github.com/mineiros-io/terraform-aws-vpc/tree/master/test
+<!-- markdown-link-check-enable -->
```
Comment (soerenmartius) [inline test/README.md:69]:
```suggestion
```

**Comment by mariux** [comment]
removed all markdown-link... felt like initial development.. so forgot to check this.. ;)

**Comment by soerenmartius** [comment]
lgtm! 8️⃣

**Pull request merged**